packages/lesswrong/lib/types/collectionTypes.ts

```diff
@@ -110,6 +110,7 @@ type CollectionOptions<N extends CollectionNameString> = {
   interfaces?: string[],
   description?: string,
   logChanges?: boolean,
+  wal?: boolean
```

Comment:
I feel like this could be named better, like `writeAheadLogged` or `postgresWal` or `postgresLogged` or something. Or, if not, it should definitely have a doc comment.